pyvista/plotting/plotting.py

```diff
@@ -6253,7 +6248,12 @@ def on_timer(iren, event_id):
         self.set_background(self._theme.background)
 
         # Set window size
-        self.window_size = window_size
+        self._window_size_unset = False
+        if window_size is None:
+            self.window_size = self._theme.window_size
+            self._window_size_unset = True
```

Comment:
My only last minute thought is to add a check here that will verify the `window_size` matches that of `DefaultTheme.window_size` this would allow users to set the `window_size` in the theme and have that control the Tram Jupyter widget size.

```suggestion
            if self.window_size == pyvista.themes.DefaultTheme().window_size
                self._window_size_unset = True
```

thoughts, @larsoner?

Though this might mess up the testing as the `TestingTheme` sets a different window size

Reply:
Done